src/daemon/rpm-ostreed-automatic.service.in

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=RPM-OSTree Automatic Update
+Documentation=man:rpm-ostree(1) man:rpm-ostreed.conf(5)
+ConditionPathExists=/run/ostree-booted
+
+[Service]
+Type=oneshot
+ExecStart=@bindir@/rpm-ostree upgrade --automatic
```

Comment:
> I wonder if it'd actually be simpler to in this case to have `--automatic` imply "journal to self".

What does "journal to self" mean here? I.e. tell the daemon to send directly to the journal rather than over D-Bus? Hmm, yeah that makes sense. Let's do this as a follow-up, maybe? (We definitely want `--quiet` too, agreed!)